qhue/qhue.py

```diff
@@ -26,6 +24,7 @@ def __init__(self, url, timeout=_DEFAULT_TIMEOUT):
         if post_username_match is not None:
             self.short_address = post_username_match.group(1)
         self.timeout = timeout
+        self.object_pairs_hook = object_pairs_hook
```

Comment:
I considered calling this `object_type`, but I figured that more people would be familiar with the `json` module's `object_pairs_hook` for when they needed to customize JSON deserialization.